tests/pytest.ini

```diff
@@ -10,3 +10,4 @@ markers =
     nightly:Tests that should be run nightly
     world_size:Change world size of individual tests in a class
     stable_diffusion:Tests that run Stable Diffusion
+    evaluation: Tests that evaluate model correctness
```

Comment:
No space after the : here?

Reply:
Yeah, it varies across the marks, I'll just make it consistent across all and remove the space.